sansa-owl-spark/src/main/resources/catalog-v001.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<catalog prefer="public" xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
+    <group id="Folder Repository, directory=, recursive=true, Auto-Update=true, version=2" prefer="public" xml:base="">
+        <uri id="Automatically generated entry, Timestamp=1568123163853" name="http://ex.com/default" uri="OWLXMLTest.owl"/>
+    </group>
+</catalog>
```

Comment:
Actually, I don't know how to do that :(

Reply:
Just add the line

catalog-v001.xml

to the `.gitignore` file which already exists in the Git repository's root directory. Then, after having deleted the file sansa-owl-spark/src/main/resources/catalog-v001.xml it will not be considered by Git anymore.